examples/ShowRecorder.h

```diff
@@ -37,7 +37,8 @@
 class ShowRecorder {
  public:
   ShowRecorder(const std::string &filename,
-               const std::vector<unsigned int> &universes);
+               const std::vector<unsigned int> &universes,
+               const std::string &channeltrigger);
```

Comment:
We use snake_case for variables (but see the other comments before diving in and fixing it):
```suggestion
               const std::string &channel_trigger);
```